scipy/optimize/_linprog_highs.py

```diff
@@ -350,12 +341,14 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
 
     options = {
         'presolve': presolve,
-        'sense': 1,  # minimization
+        'sense': HIGHS_OBJECTIVE_SENSE_MINIMIZE,
         'solver': solver,
         'time_limit': time_limit,
-        'message_level': MESSAGE_LEVEL_MINIMAL * disp,
+        'highs_debug_level': MESSAGE_LEVEL_NONE,
         'dual_feasibility_tolerance': dual_feasibility_tolerance,
         'ipm_optimality_tolerance': ipm_optimality_tolerance,
+        'log_to_console': disp,
```

Comment:
I can't remember what was going on here - we were never able to get HiGHS output in python `stdout`? Using Spyder, I can't see output from HiGHS on master or this branch. It shows up in the terminal that is running Spyder.

Reply:
I remember discussing this in the original HiGHS PR, but I also don't recall if we ever made this work in Spyder.  If it's an issue on current master, then let's make a new issue for it and consider it out of scope for this PR (i.e. keep the status quo on how console logging is currently handled)

Reply:
Yes, I think it's in master, too. Resolving here.

Reply:
Yes, I think it's in master, too. Resolving here.